luaui/Widgets/gui_gridmenu.lua

```diff
@@ -2301,84 +2458,148 @@ function widget:DrawScreen()
 
 		checkGuishaderBuilders()
 		handleButtonHover()
+
+		if currentlyBuildingRectID then
+			drawBuildProgress(cellRects[currentlyBuildingRectID])
+		end
 	end
 end
 
+-- function widget:DrawWorld()
+-- 	-- Avoid unnecessary overhead after buildqueue has been setup in early frames
+-- 	if Spring.GetGameFrame() > 0 then
+-- 		widgetHandler:RemoveWidgetCallIn("DrawWorld", self)
+-- 		return
+-- 	end
+--
+-- 	if not isPregame then
+-- 		return
+-- 	end
+--
+-- 	if startDefID ~= Spring.GetTeamRulesParam(myTeamID, "startUnit") then
+-- 		startDefID = Spring.GetTeamRulesParam(myTeamID, "startUnit")
+-- 		redraw = true
+-- 	end
+--
+-- end
```

Comment:
If this code is never going to be needed, it can just be deleted.

Reply:
Good call, this is a remnant of the refactoring, I'll make sure I delete If I don't need when I remove from draft